org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/commons/TextDocuments.java

```diff
@@ -26,21 +26,13 @@
  */
 public class TextDocuments implements ITextDocumentFactory {
 
-	private boolean incremental;
+	private final boolean incremental;
 
 	private final Map<String, TextDocument> documents;
 
 	public TextDocuments() {
 		documents = new HashMap<>();
-	}
-
-	/**
-	 * Set the incremental support.
-	 * 
-	 * @param incremental
-	 */
-	public void setIncremental(boolean incremental) {
-		this.incremental = incremental;
+		incremental = true; // Quick fix to set incremental 

```

Comment:
Please remove this hard coded incremental support and let's the user to set incremental support as it's a major changed. Once we will have good feedback from users, we will switch to incremental support every time.